helm.el

```diff
@@ -3175,13 +3176,22 @@ and sort on basename of candidates."
                      (scr1 (car data1))
                      (scr2 (car data2)))
                 (cond ((= scr1 scr2)
-                       (< len1 len2))
+                       (unless preserve-tie-order
+                         (< len1 len2)))
                       ((> scr1 scr2)))))))))
 
 (defun helm-fuzzy-matching-default-sort-fn (candidates _source &optional use-real)
   "Default `filtered-candidate-transformer' to sort candidates in fuzzy matching."
   (helm-fuzzy-matching-default-sort-fn-1 candidates use-real))
 
+(defun helm-fuzzy-matching-sort-fn-preserve-ties-order (candidates _source &optional use-real)
+  "`filtered-candidate-transformer' to sort candidates in fuzzy matching, preserving order of ties.
```

Comment:
:man_shrugging: I like to keep interfaces (i.e. public fns, vars, etc.) as small as possible. Makes code easier to refactor.